samples/nrf9160/nrf_cloud_rest_device_message_tol2/Kconfig

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "nRF Cloud REST Device Message Sample"
+
+config REST_DEVICE_MESSAGE_SAMPLE_VERSION
+	string "The sample's version string"
+	default "1.0.0"
+
+config REST_DEVICE_MESSAGE_LTE_LED_NUM
+	int "The LED number to use for indicating LTE connection success"
+	default 0
+
+config REST_DEVICE_MESSAGE_SEND_LED_NUM
+	int "The LED number to use for indicating outgoing REST request"
+	default 1
+
+config REST_DEVICE_MESSAGE_BUTTON_EVT_NUM
+	int "Button number to use for device interactions"
+	default 1
+
+config REST_DEVICE_MESSAGE_DO_JITP
+	bool "Ask to perform just-in-time provisioning (JITP) via REST on startup"
```

Comment:
Since JITP is not supported in your sample, you should remove all mention of it in this file as well as in others, like README.rst.

Reply:
I'll delete all the mentions about JITP. Thank you.